tests/test_bot.py

```diff
@@ -533,6 +533,29 @@ async def post(url, request_data: RequestData, *args, **kwargs):
             123, "text", api_kwargs={"unknown_kwarg_1": 7, "unknown_kwarg_2": 5}
         )
 
+    async def test_get_updates_deserialization_error(self, bot, monkeypatch, caplog):
+        async def faulty_do_request(*args, **kwargs):
+            return (
+                HTTPStatus.OK,
+                b'{"ok": true, "result": [{"update_id": "1", "message": "unknown_format"}]}',
+            )
+
+        monkeypatch.setattr(HTTPXRequest, "do_request", faulty_do_request)
+
+        bot = ExtBot(get_updates_request=HTTPXRequest(), token=bot.token)
+
+        with caplog.at_level(logging.CRITICAL), pytest.raises(AttributeError):
+            await bot.get_updates()
```

Comment:
does the bot not need initializing? If so, it's better to use `PytestExtBot` instead